Description: Find trace-cmd

 We don't have trace-cmd pre-installed as its from the same source. From the
 next version trace-cmd and kernelshark will be separate source and this patch
 hardcoding the trace-cmd location can be removed.

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Forwarded: not-needed

---

--- trace-cmd-2.9.2.orig/kernel-shark/src/KsCaptureDialog.cpp
+++ trace-cmd-2.9.2/kernel-shark/src/KsCaptureDialog.cpp
@@ -554,10 +554,7 @@ KsCaptureDialog::KsCaptureDialog(QWidget
 	connect(&_captureCtrl._closeButton,	&QPushButton::pressed,
 		this,				&KsCaptureDialog::close);
 
-	if (!KsUtils::isInstalled())
-		captureExe = QString(_INSTALL_PREFIX) + QString("/bin/trace-cmd");
-	else
-		captureExe = TRACECMD_EXECUTABLE;
+	captureExe = QString(_INSTALL_PREFIX) + QString("/bin/trace-cmd");
 
 	_captureProc.setProgram(captureExe);
 
